packages/language/test/test-builder.ts

```diff
@@ -480,13 +480,29 @@ export class TestBuilder {
     return this;
   }
 
-  noDiagnosticsExcept(regex: RegExp[], label?: Label): TestBuilder {
+  noDiagnosticsExcept(
+    exceptions: RegExp[] | string[],
+    label?: Label,
+  ): TestBuilder {
+    if (exceptions.length === 0) {
+      return this;
+    }
+    if (typeof exceptions[0] === "string") {
+      this.noDiagnosticsExcept(
+        (exceptions as string[]).map(
+          (s) => new RegExp(s.replace(/[.*+?^${}()|[\]\\]/g, "\\$&")),
+        ),
+        label,
+      );
+      return this;
+    }
```

Comment:
Suggestion: I think it makes more sense to convert all of them to regex and continue.
```suggestion
    if (typeof exceptions[0] === "string") {
      exceptions = (exceptions as string[]).map(escapeRegExp);
    }
```

Reply:
👍 makes sense. thanks!